build/configs/rtl8720e/scripts/rlx8720e_img2_is.ld

```diff
@@ -69,7 +69,7 @@ MEMORY
 	KM4_FLOADER_RAM_S (rwx) :               ORIGIN = 0x3000A020, LENGTH = 0x30020000 - 0x3000A020	/* KM4 ImgTool Flash Loader RAM: 88k */
 	KM4_BD_RAM_NSC (rwx) :                  ORIGIN = 0x6015B020, LENGTH = 0x6015C000 - 0x6015B020	/* KM4 BD RAM NSC: 4k */
 	KM4_BD_RAM_ENTRY (rwx) :                ORIGIN = 0x6015C000, LENGTH = 0x60160000 - 0x6015C000	/* KM4 BD RAM ENTRY: 16k */
-	KM4_BD_RAM_S (rwx) :                    ORIGIN = 0x70160020, LENGTH = 0x70180000 - 0x70160020	/* KM4 BD RAM S: 128k */
+	KM4_BD_RAM_S (rwx) :                    ORIGIN = 0x70160020, LENGTH = 0x7018D000 - 0x70160020	/* KM4 BD RAM S: 128k */

```

Comment:
@ZhenBei-Sin Could you update the value in the comment? It's still 128k.
@fengyunlai Could you check your peripheral change as well? PR #5560 did not update it.